includes/forms/template.php

```diff
@@ -648,6 +650,8 @@ class="give-input required"
 					name="give_first"
 					placeholder="<?php _e( 'First Name', 'give' ); ?>"
 					id="give-first"
+					pattern="[A-Za-z]+"
```

Comment:
Ditch this because we're likely moving away from HTML5 validation in the future.